meta-iotlab/recipes-devtools/openocd/openocd-ti.bb

```diff
@@ -0,0 +1,76 @@
+DESCRIPTION = "TI's fork of OpenOCD Free and Open On-Chip Debugging, In-System Programming and Boundary-Scan Testing"
+HOMEPAGE = "git://git.ti.com/sdo-emu/openocd.git"
+SECTION = "utils"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+
+PR = "r1"
+
+DEPENDS = ""
+DEPENDS += "libftdi"
+DEPENDS += "libusb1"
+DEPENDS += "jimtcl"
+DEPENDS += "hidapi"
+DEPENDS += "texinfo-native"
+
+## ##################################################
+## ##################################################
+
+# Use git version 0.10.0: issues when using source code
```

Comment:
The comment is bit misleading: the git version is also source code. I guess it means "issues when using latest master version" or something ?

Reply:
sorry, it's a copy paste from openocd-0.10.0.bb, the comment applied there, but not here. will fix